### ISSUE TITLE
Add term periumbilical depression

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10447]
**Preferred term label:**
Periumbilical depression

**Synonyms**

**Definition (free text, please give PubMed ID)**
A depression of the tissue that immediately surrounds the umbilicus.

![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/228092cf-5ce4-4a31-918c-1df30ac1bf7a)
Source: PMID: 21035106

Comment: The umbilicus is also known as the belly button or the navel.

**Parent term (use hpo.jax.org/app)**
Abnormal umbilicus morphology HP:0001551

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
3MC syndrome 1 (OMIM257920), PMID: 16096999, PMID: 21035106, PMID: 18266249

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000808
- label: Periumbilical depression
- parents: [Abnormal umbilicus morphology (HP:0001551)](https://hpo.jax.org/app/browse/term/HP:0001551)
- pmid: PMID:16096999
- def: The tissue that immediately surrounds the umbilicus is located below the normal plane of the abdomen.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10447
- synonyms:
